source/template.yaml

```diff
@@ -585,6 +559,87 @@ Resources:
     Metadata:
       BuildMethod: ruby2.7
 
+  # update with ssm parameter
```

Comment:
what does this comment mean?

Reply:
I have created a new SSM parameter in identity-devops with that contains the arn for the notification topics.  Once that gets deployed I am going to update the template to use the parameter value instead of hard coding the topic name.